docs(tickets): update publish ticket content (#209)
AlaskaAirlines/WC-Generator: template/.github/ISSUE_TEMPLATE/publish_component.md

## Specific steps required to publish this component.

> 1. __Design & Planning__

Reviewer: It's not in this PR, but please checkout and update line 3, there are some typos. 

```
about: Create a ticket to drive compliance before publishing a a new componet to Auro Design System
```

    - [ ] Reviewed [AuroLabs documentation](http://auro.alaskaair.com/)
    - [ ] Reviewed [Auro's Contributing Guidelines](http://auro.alaskaair.com/contributing)
    - [ ] Reviewed [Auro's CSS conventions](http://auro.alaskaair.com/webcorestylesheets/conventions) and how to apply CSS to custom elements
    - [ ] Design team engaged in designing the new component
    - [ ] Design work integrated in the Auro Design System UI Kit
1. __Development__
    - [ ] Addresses all design and UX as proposed in the design comp
    - [ ] A11y in regards to role and live notifications need to be coded into the final WC
    - [ ] All style code follows [Auro's Style Guide](http://auro.alaskaair.com/webcorestylesheets/guidelines)
    - [ ] Automated tests created
1. __Documentation__
    - `demo.md`
      - [ ] Component properly described

>       - [ ] All component `attributes` listed and described

Reviewer: ```suggestion
      - [ ] Custom element's full API is documented
```

      - [ ] Component use case(s) outlined
      - [ ] `Do Not` section outlines implementation scenarios to avoid
      - Example Section
        - [ ] Basic example of simplest use case
        - [ ] Examples showing use of each API attribute
        - [ ] All examples include rendered example and matching code sample
    - `readme.md`
      - [ ] Component properly described
      - [ ] Bundled version info
      - [ ] Use cases described

>       - [ ] `API Code Examples` updated to match API with working example code

Reviewer: For maintenance, I am really leaning toward not putting too many examples in the README. As things change, this is always forgotten. 

I'd much prefer that we link to the Docsite whenever possible for stuff like this.

Author: The readme.md template has a single code example. This checkbox is to make sure that example is valid within the API definition.

If we want to change the current template content let's open a separate ticket for that. I would be in favor the readme.api simply referencing/linking the demo examples via a link and remove the code example.

    - [ ] `API.md` document created and matches current API
1. __Testing__
    - [ ] Component built using latest version of [WC-Generator](http://auro.alaskaair.com/getting-started/developers/generator/install)
    - [ ] All automated tests pass and coverage threshold requirements are met

>     - [Browser compatibility](http://auro.alaskaair.com/support/browsersSupport) - desktop and mobile versions
>       - [ ] Chrome
>       - [ ] Firefox
>       - [ ] Edge (79 +)
>       - [ ] Safari

Reviewer: ```suggestion
    - [ ] [Browser compatibility](http://auro.alaskaair.com/support/browsersSupport) - desktop and mobile versions
```

All the browsers are listed in the link, I think we can safely assume that all the browsers were reviewed.

    - [Accessibility](http://auro.alaskaair.com/a11y-statement) - One or more of the following
      - [ ] Axe / Google Lighthouse (baseline standard)
      - [ ] Screenreader (determine usability of experience via screenreader)
      - [ ] Keyboard use (a user must be able to navigate to key parts via a keyboard)
      - [ ] Accessibility should be verified against all browsers
    - [ ] Bundled versions tested
1. __Review__

> 

Reviewer: Delete white space

    When the component is ready for a final review it is the responsibility of the contributor to coordinate the following reviews:

>     - [ ] Review demo with the designer

Reviewer: ```suggestion
    - [ ] Reviewed demo with the designer
```

>     - [ ] Review demo with Auro Team

Reviewer: ```suggestion
    - [ ] Reviewed demo with Auro Team
```

    - [ ] All work is reviewed by Auro team
1. __Publishing__

> 

Reviewer: Delete white space

    - [ ] Removed all references to `auroLabs` from the repository

>     - [ ] Component properly named (e.g. `auro-widget`)

Reviewer: ```suggestion
    - [ ] Component properly named,  e.g. `auro-[name]`
```

This is how it's referenced in all the other docs, we are not using `widget` as far as I know as a term.

    - [ ] GitHub Repository properly named
    - Updated `readme.md`
      - [ ] Install information updated to published namespace
      - [ ] `Define dependency in project component` updated to published namespace
      - [ ] `Bundle example code` updated to published namespace

>     - [ ] Demo page included in the Auro Doc Site

Reviewer: ```suggestion
    - [ ] PR prepared to include demo page in the Auro Doc Site
```

This can't be completed until the release is complete.
